### PR TITLE
made community newsletter form same

### LINKED ISSUE #987: Community Newsletter component should be consistent
### Description
Community Newsletter component should be consistent across all screens

image
image
image

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Adi-ty**
Signed-off-by: Aditya <adityasinghboss1234@gmail.com>

This PR makes the community newsletter form to have similar casing in all pages please review and merge this PR or suggest changes if needed

This PR fixes #987 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Adi-ty** [comment]
@GaganpreetKaurKalsi please review and merge or suggest changes if needed

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @Adi-ty. When choosing a consistent format, it should either be:

"Community Newsletter" or all uppercase, but not all lowercase.

**Comment by Adi-ty** [comment]
Ok I'll fix it

**Comment by Adi-ty** [comment]
@leecalcote  I have Changed it to "Community Newsletter" 

/#getting-started
image

/programs
image

/subscribe
image

**Comment by Adi-ty** [comment]
I just saw that the community newsletter form on `/programs` page has an extra 50px margin below the heading should I remove that also @leecalcote

**Review by GaganpreetKaurKalsi — CHANGES_REQUESTED** [review]
Sure, remove that extra margin. Also capitalise `Subscribe` as well.

**Comment by Adi-ty** [comment]
OK will do it

**Comment by Adi-ty** [comment]
Has done all the changes required @leecalcote @GaganpreetKaurKalsi

**Review by GaganpreetKaurKalsi — DISMISSED** [review]
Looks good now 👍 Thanks @Adi-ty 💯

**Comment by Adi-ty** [comment]
@leecalcote please review this PR.

**Comment by leecalcote** [comment]
Will you also have a look into the mismatched border radius and white stroke/border?

**Comment by Adi-ty** [comment]
Should I make changes for every border radius including the whole form, various inputs and subscribe button and also can you elaborate on white stroke/border.

**Comment by Adi-ty** [comment]
Ok I am making every form have a border radius of 10px and also, I will remove the white outline on subscribe button on one of the page.

**Comment by Adi-ty** [comment]
@leecalcote I have made all the changes please review and merge

**Review by GaganpreetKaurKalsi — APPROVED** [review]
LGTM 👍

File: _includes/homePage.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
       button {
         color: white;
         background: transparent;
-        outline: 1px solid white;
+        <!-- outline: 1px solid white; -->
```
Comment (leecalcote) [inline _includes/homePage.html:61]: ```suggestion```

File: _sass/forms.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @import "./mixins/transitions.scss";
 .form-container{
 	background: var(--background-grey-secondary);
-	border-radius: 7px;
+	border-radius: 10px;
```
Comment (leecalcote) [inline _sass/forms.scss:4]:
```suggestion
	border-radius: 0.625rem;
```

File: _includes/homePage.html (hunk 1 above)
Comment (leecalcote) [reply _includes/homePage.html:61]:
```suggestion
```

**Review by leecalcote — APPROVED** [review]
Thanks, @Adi-ty

**Pull request merged**